cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -166,12 +166,24 @@ set(CMAKE_SYSTEM_INCLUDE_PATH "${EMSCRIPTEN_ROOT_PATH}/system/include")
 # but if these are adjusted, ${CMAKE_ROOT}/Modules/CheckIncludeFile.cmake will fail, since it depends on being able to compile output files with predefined names.
 #SET(CMAKE_LINK_LIBRARY_SUFFIX "")
 #SET(CMAKE_STATIC_LIBRARY_PREFIX "")
-#SET(CMAKE_STATIC_LIBRARY_SUFFIX ".bc")
```

Comment:
granted I know nothing about cmake ;) but to me this looks like you are removing a line that defaulted the suffix to `.bc`, and later below do that only if an option is set. So it appears to change the default behavior? I'm sure i'm missing something...